x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality_panel/constants.ts

```diff
@@ -57,3 +57,13 @@ export const EMPTY_METADATA: PartitionedFieldMetadata = {
   incompatible: [],
   sameFamily: [],
 };
+
+export const EMPTY_PLACEHOLDER = '--';
+
+export const ECS_FIELD_REFERENCE_URL =
+  'https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html';
+
+/** The documentation link shown in the `Data Quality` dashboard */
+export const ECS_REFERENCE_URL = 'https://www.elastic.co/guide/en/ecs/current/ecs-reference.html';
+export const MAPPING_URL =
+  'https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html';
```

Comment:
Nothing to do with this change, just want to make a note of a potential task:
As we are going to have ESS and serverless docs separately.
It would be great for us to review all the doc links we have in security solution, and retrieve them with `getDocLinks` e.g.: https://github.com/elastic/kibana/pull/189075/files#diff-aa872c6e8f0a6d2bc64ca2721a46a81cad6b6c7b62112b50849f2c05f3680e9fR33
so it takes care of the Kibana version and Kibana type.

Reply:
didn't know about the tool, thanks